capstone-frontend/src/components/LeftSideBar.jsx

```diff
@@ -1,122 +1,454 @@
+              <InputBase
+                autoComplete="off"
+                id="search-input"
+                placeholder="Search…"
+                classes={{
+                  root: classes.inputRoot,
+                  input: classes.inputInput,
+                }}
+                inputProps={{ 'aria-label': 'search' }}
+              />
+            </form>
+          </div>
+          <div className={classes.grow} />
+
+          <div className={classes.sectionDesktop}>
+            <Logout toggleSignIn={props.toggleSignIn} />
+          </div>
+        </Toolbar>
+      </AppBar>
+      <Drawer
+        variant="permanent"
+        className={clsx(classes.drawer, {
+          [classes.drawerOpen]: open,
+          [classes.drawerClose]: !open,
+        })}
+        classes={{
+          paper: clsx({
+            [classes.drawerOpen]: open,
+            [classes.drawerClose]: !open,
+          }),
+        }}
+      >
+        <div className={classes.toolbar}>
+
+          <div className={classes.sectionDesktop}>
+            <Avatar
+              alt={profileObj.name}
+              src={profileObj.imageUrl} />
+
+            <Typography
+              noWrap
+              gutterBottom
+              className="avatar-text-style"
+              align="center"
+              color="textPrimary"
+              variant="h6" >
+              {profileObj.name}
+            </Typography>
+          </div>
+
+          <div className={classes.sectionMobile.concat(' margin-auto')}>
+            <Avatar
+              className={classes.small}
+              alt={profileObj.name}
+              src={profileObj.imageUrl} />
+
+            <Typography
+              noWrap
+              gutterBottom
+              className="avatar-text-style"
+              align="center"
+              color="textPrimary"
+              variant="h6" >
+              {profileObj.name}
+            </Typography>
+          </div>
+
+          <IconButton onClick={handleDrawerClose}>
+            {theme.direction === 'rtl' ? <ChevronRightIcon /> : <ChevronLeftIcon />}
+          </IconButton>
+
+        </div>
+
+        <Divider />
+        <List>
+          <Link to="/" className="remove-link-style" id="home-link">
+            <ListItem button>
+
+              <ListItemIcon>
+                <HomeIcon />
+              </ListItemIcon>
+              <ListItemText primary="Home" />
+
+            </ListItem>
+          </Link>
+
+          <Link to="/browse" className="remove-link-style" id="browse-link">
+            <ListItem button>
+
+              <ListItemIcon>
+                <SearchIcon />
+              </ListItemIcon>
+              <ListItemText primary="Search" />
+
+            </ListItem>
+          </Link>
+
+          <Link to="/myreads" className="remove-link-style" id="mybooks-link">
+            <ListItem button >
+
+              <ListItemIcon>
+                <MenuBookIcon />
```

Comment:
Is it possible to render the names of the tabs rather than icons? I don't think the icons are super clear/obvious to users which tab opens which page.

Reply:
I found myself struggling to open the right pages without seeing text

Reply:
We could change the icon later down the road. The point of this LeftSideBar is that is a MiniDrawer, so you can expanded and minimize it.

Reply:
I don't think names would fit the style of the drawer. We could start the Drawer open by default if you want?